otherlibs/Makefile.otherlibs.common

```diff
@@ -44,8 +43,6 @@ MKLIB=$(OCAMLRUN) $(ROOTDIR)/tools/ocamlmklib
 # Variables that can be defined by individual libraries,
 # but have sensible default values:
 COBJS ?=
-EXTRACFLAGS ?=
-EXTRACPPFLAGS ?=
```

Comment:
Just FTR, `EXTRACFLAGS` was an incorrectly named variable added in 4.02 for the win32unix `Makefile` and corrected with `EXTRACPPFLAGS` in #9518. They're both obsolete since the merging of the unix/win32unix directories in #11160